Dockerfile

```diff
@@ -1,26 +1,25 @@
 # Use an official Python base image from the Docker Hub
 FROM python:3.11-slim
 
-# Install git
-RUN apt-get -y update
-RUN apt-get -y install git
+# Install packages
+RUN apt-get update && apt-get install -y curl jq git
 
 # Set environment variables
 ENV PIP_NO_CACHE_DIR=yes \
     PYTHONUNBUFFERED=1 \
     PYTHONDONTWRITEBYTECODE=1
 
-# Create a non-root user and set permissions
-RUN useradd --create-home appuser
-WORKDIR /home/appuser
-RUN chown appuser:appuser /home/appuser
-USER appuser
+# Install the required python packages globally
+ENV PATH="$PATH:/root/.local/bin"
+COPY requirements-docker.txt .
+RUN pip install --no-cache-dir -r requirements-docker.txt
 
-# Copy the requirements.txt file and install the requirements
-COPY --chown=appuser:appuser requirements-docker.txt .
-RUN pip install --no-cache-dir --user -r requirements-docker.txt
+# Create a non-root user
```

Comment:
Isn't appuser already in the base container?

Reply:
Not as far as I can find, no.